Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "cargo-edit"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
`cargo update` will also, well, update dependencies. Which we should only ever do with careful testing.